frontend/src/sass/core/_mixins.scss

```diff
@@ -4,3 +4,8 @@
   -o-transition: all $duration $effect;
   transition: all $duration $effect;
 }
+
+@mixin h2 {
```

Comment:
Why are we redefining this here instead of just using the one provided by USWDS?

Reply:
I can either import the 252 lines of _utilities.scss from uswds which is currently not imported, or I could add 3 lines of code. Was trying to keep it easy to understand for future devs. If you prefer I can import '../node_modules/uswds/src/stylesheets/core/_utilities.scss' instead.

Reply:
I hear where you're coming from, here are my thoughts. USWDS is a heavy dependency for this application, it is a requirement that devs learn to use it so we shouldn't shy away from forcing them to have an understanding of the dependency. To me, this is akin to using a feature provided by Express vs. duplicating it ourselves bc we don't want to burden the developer with having to learn that framework.

This also helps in the case of future upgrades, for example, what if we upgrade to USWDS v2 and all the header styles change slightly, our `h2` mixin would be out of sync with that. We obviously have deviations from USWDS currently in the app, but I would rather minimize those as much as possible.

On a related note, while I'm generally in favor of performance optimizations, the amount of CSS we have is not currently bottleneck so I wouldn't worry about minimizing it for now.